src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/ObjectWriter.cs

```diff
@@ -88,10 +88,13 @@ public void EmitSymbolRef(string symbolName, int size, bool isPCRelative, int de
         }
 
         [DllImport(NativeObjectWriterFileName)]
-        private static extern void EmitFrameInfo(IntPtr objWriter, string methodName, int startOffset, int endOffset, int blobSize, byte[] blobData);
-        public void EmitFrameInfo(string methodName, int startOffset, int endOffset, int blobSize, byte[] blobData)
+        private static extern void EmitFrameInfo(IntPtr objWriter, string methodName, int startOffset, int endOffset, int blobSize, byte[] blobData,
+                                                 string personalityFunctionName, int LSDASize, byte[] LSDA);
+        public void EmitFrameInfo(string methodName, int startOffset, int endOffset, int blobSize, byte[] blobData,
+                                  string personalityFunctionName = null, int LSDASize = 0, byte[] LSDA = null)
```

Comment:
What does the object writer do with null?  Will it emit the __CxxFrameHandler3 personality routine like it used to?  If not, don't we need to pass something to make the unwind info consistent?  As I understand it the JIT always emits the UNW_EHANDLER | UNW_UHANDLER in the flags, like it does for CLR.